.github/workflows/main.yml

```diff
@@ -6,7 +6,23 @@ name: CI
 on:
   # Triggers the workflow on push or pull request events but only for the develop branch
   push:
+    branches:
+      - main
+      - dev # branch used for testing incremental builds
```

Comment:
We don't have a dev branch on the repo, so I think we can remove this

Reply:
resolved as per https://github.com/geoCML/geocml-base-deployment/pull/61/commits/4475f517d0f5e1704246ce2cb096ebf2cbeae8e1